ddtrace/commands/ddtrace_run.py

```diff
@@ -16,54 +27,89 @@
 log = logging.getLogger(__name__)
 
 USAGE = """
-Execute the given Python program after configuring it to emit Datadog traces.
-
-Append command line arguments to your program as usual.
+Execute the given Python command after configuring it to emit Datadog traces.
+The Datadog profiler can also be enabled.
 
-Usage: ddtrace-run <my_program>
-"""  # noqa: E501
 
-
-def _ddtrace_root():
-    from ddtrace import __file__
-    return os.path.dirname(__file__)
+Examples
+ddtrace-run python app.py
+ddtrace-run uwsgi app.py
+ddtrace-run gunicorn myproject.wsgi
+"""
 
 
 def _add_bootstrap_to_pythonpath(bootstrap_dir):
     """
     Add our bootstrap directory to the head of $PYTHONPATH to ensure
     it is loaded before program code
     """
-    python_path = os.environ.get('PYTHONPATH', '')
+    python_path = os.environ.get("PYTHONPATH", "")
 
     if python_path:
-        new_path = '%s%s%s' % (bootstrap_dir, os.path.pathsep, os.environ['PYTHONPATH'])
-        os.environ['PYTHONPATH'] = new_path
+        new_path = "%s%s%s" % (bootstrap_dir, os.path.pathsep, os.environ["PYTHONPATH"])
+        os.environ["PYTHONPATH"] = new_path
     else:
-        os.environ['PYTHONPATH'] = bootstrap_dir
+        os.environ["PYTHONPATH"] = bootstrap_dir
 
 
 def main():
-    if len(sys.argv) < 2 or sys.argv[1] == '-h':
-        print(USAGE)
-        return
-
-    log.debug('sys.argv: %s', sys.argv)
-
-    root_dir = _ddtrace_root()
-    log.debug('ddtrace root: %s', root_dir)
-
-    bootstrap_dir = os.path.join(root_dir, 'bootstrap')
-    log.debug('ddtrace bootstrap: %s', bootstrap_dir)
+    parser = argparse.ArgumentParser(
+        description=USAGE,
+        prog="ddtrace-run",
+        usage="ddtrace-run <your usual python command>",
```

Comment:
```suggestion
        usage="ddtrace-run <script>",
```

? wondering if "your usual python command" is too verbose for this?

I could also be over thinking as well

Reply:
By saying <script> it sounds like you no longer need to include the python executable in the command (the most common misusage of ddtrace-run)

Reply:
`ddtrace-run <python script.py>`?

idk why `<your usual python command>` feels oddly verbose and not specific enough?

we can also example the examples in the help text as well

Reply:
Agreed, happy to consider an alternative that is more succinct 😆

> we can also example the examples in the help text as well

![image](https://user-images.githubusercontent.com/6321485/94172589-ad896680-fe60-11ea-8c64-a58d85f8798c.png)

Reply:
> ddtrace-run <python script.py>?

This also doesn't work because it doesn't cover uwsgi/gunicorn